pkg/nfd-worker/nfd-worker.go

```diff
@@ -546,21 +567,21 @@ func getFeatureLabels(source source.FeatureSource, labelWhiteList regexp.Regexp)
 		// Validate label name.
 		errs := validation.IsQualifiedName(nameForValidation)
 		if len(errs) > 0 {
-			stderrLogger.Printf("Ignoring invalid feature name '%s': %s", label, errs)
+			klog.Warningf("Ignoring invalid feature name '%s': %s", label, errs)
```

Comment:
Should we add some sort of debug lvl to this? or as a warning is just ok?

Reply:
Basically for all warnings, is just a question, not a blocker, but I wonder if they could be moved to a debug lvl or something.

Reply:
The thinking behind the warning is that in a properly configured setup you should not get these. I.e. the above looks like the use tries to create label X but it is not possible because the name does not qualify as a K8s label.